Workbooks/WaasUpdateInsights/DeliveryOptimization/DeliveryOptimization.workbook

```diff
@@ -0,0 +1,366 @@
+              "durationMs": 86400000
+            },
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          },
+          {
+            "id": "f56d6943-b8f5-449e-aaaf-06100a59e8b5",
+            "version": "KqlParameterItem/1.0",
+            "name": "_WorkbookContext",
+            "label": "Workbook Context",
+            "type": 1,
+            "description": "Whether workbook is opened standalone or as part of the Hub.",
+            "isRequired": true,
+            "query": "print(\"Focused\")",
+            "isHiddenWhenLocked": true,
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "name": "parameters - 11"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "## Delivery Optimization Status\n---\n**Delivery Optimization** (DO) is a Windows 10 feature that can be used to reduce bandwidth consumption by sharing the work of downloading updates among multiple devices in your environment. You can use DO in conjunction with many other deployment methods, but it is a cloud-managed solution, and access to the DO cloud services is a requirement. To learn more about DO, view the documentation on [**Configuring Delivery Optimization for Windows 10 updates**](https://docs.microsoft.com/en-us/windows/deployment/update/waas-delivery-optimization). Note: This section does not include Insider devices.\n\n**Important Terms**\n\nA **Peer** is a device in the solution.\n\n**Peering ON** refers to devices in which DO Peer-to-peer is enabled in one of the following modes: LAN (1), Group (2), or Internet (3).\n\n**Peering OFF** refers to devices in which DO Peer-to-peer is disabled, set to one of the following modes: HTTP Only (0), Simple Mode (99), or Bypass (100).\n\n**Bandwidth Savings** represents the percentage of bandwidth that was downloaded from LAN or Group peers out of the total consumed bandwidth.\n\n**Configurations** are based on the DownloadMode configuration set via MDM, GP, or end-user via the user interface."
+      },
+      "customWidth": "100",
+      "name": "text - 2"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "let _SnapshotTime = datetime({_SnapshotTime});\r\n\r\nWUDOAggregatedStatus| where TimeGenerated >= _SnapshotTime\r\n| where ContentType in (\"Feature and Flight Updates\" , \"Quality Updates\"  ) | summarize BytesFromPeers = round((sum(0.0 + BytesFromPeers + BytesFromGroupPeers)/ sum(BytesFromPeers + BytesFromGroupPeers+BytesFromCDN)) * 100.0 ,2)",
+        "size": 4,
+        "title": "% bandwidth savings in Feature and Quality Updates over the last 28 days",
+        "timeContext": {
```

Comment:
there seem to be at  least a couple other places that seem to have time range set to 24h instead of "Set in query" that you'll want to verify?

Reply:
Fixed, we are setting the time in the query itself.